body/stretch_body/robot_params_RE2V0.py

```diff
@@ -428,7 +428,7 @@
             'NonDXLStatusThread_sentry_downrate_int': 2,
             'NonDXLStatusThread_trajectory_downrate_int': 2},
         'tool': 'tool_stretch_gripper',
-        'use_collision_manager': 1,
+        'use_collision_manager': 0,
```

Comment:
Did you mean to turn off the collision manager in this PR?

Reply:
We did not mean to turn the collision manager off and thanks for catching. This change has been reverted.